src/components/tabs/services-tabs.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TabList, Tab, TabPanels } from '@reach/tabs';
+import { TabPanel, Tabs } from '@reach/tabs';
+import { up } from '../breakpoint/breakpoint';
+import '@reach/tabs/styles.css';
+
+const StyledUl = styled.ul`
+  padding: 0;
+  list-style-position: inside;
+`;
+
+const StyledLi = styled.li`
+  margin-bottom: 12px;
+`;
+
+const StyledTabList = styled(TabList)`
+  background: none;
+  padding-bottom: 18px;
+  overflow-x: scroll;
+  transform: translateX(-14px);
+  white-space: nowrap;
+  width: 100vw;
```

Comment:
the 100vw (and the transform) are there so that the tabs stretch the full page width on mobile as in the design. if we change it to 100% or remove it, the mobile tabs look like this:

![Screenshot 2020-06-19 at 13 42 00](https://user-images.githubusercontent.com/31735645/85129278-1eeac500-b233-11ea-9810-fdc94c121c85.png)

vs design: 

<img width="200" alt="Screenshot 2020-06-19 at 13 45 37" src="https://user-images.githubusercontent.com/31735645/85129307-29a55a00-b233-11ea-88cd-1aaefe488f3a.png">

what's the best way to implement that design, without the vw?